e2e-tests/contentful/snapshots.js

```diff
@@ -1,58 +1,58 @@
ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=400&h=600&q=50 400w,\n//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=600&h=900&q=50 600w\"\n          sizes=\"(max-width: 200px) 100vw, 200px\" /><img loading=\"lazy\"\n          sizes=\"(max-width: 200px) 100vw, 200px\"\n          srcset=\"//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=50&h=75&q=50 50w,\n//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=100&h=150&q=50 100w,\n//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=200&h=300&q=50 200w,\n//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=300&h=450&q=50 300w,\n//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=400&h=600&q=50 400w,\n//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=600&h=900&q=50 600w\"\n          src=\"//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg?w=200&q=50\"\n          alt=\"\"\n          style=\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\" />\n      </picture>\n    </noscript>\n  </div>\n  <p></p>\n  <hr>\n  <p>The languages only differ in:</p>\n  <ol>\n    <li>\n      <p>their grammar</p>\n    </li>\n    <li>\n      <p>their pronunciation</p>\n    </li>\n    <li>\n      <p>their most common words</p>\n    </li>\n    <li>\n      <p><span data-cy-id=\"inline-text\">[Inline-ContentfulText]\n          <!-- -->Text: Short\n          <!-- -->:\n          <!-- -->The quick brown fox jumps over the lazy dog.</span></p>\n    </li>\n  </ol>\n  <p>Everyone realizes why a new common language would be desirable: one could\n    refuse to pay expensive translators.</p>\n  <p><code>{\n      \"userId\": 1,\n      \"id\": 1,\n      \"title\": \"delectus aut autem\",\n      \"completed\": false\n      }</code></p>\n  <p>To achieve this, it would be necessary to have uniform grammar,\n    pronunciation and more common words.</p>\n  <p data-cy-id=\"location\">[ContentfulLocation] Lat:\n    <!-- -->52.51627\n    <!-- -->, Long:\n    <!-- -->13.3777</p>\n  <blockquote>\n    <p>If several languages coalesce, the grammar of the resulting language is\n      more simple and regular than that of the individual languages.</p>\n  </blockquote>\n  <p>The new common language will be more simple and regular than the existing\n    European languages. It will be as simple as Occidental; in fact, it will be\n  </p>\n  <hr>\n</div>"
```

Comment:
I guess these changes are due to the updated `gatsby-plugin-image`? probably @ascorbic

Reply:
Looks like it is the same thing - the new snapshot is the HTML after SSR while the old one is after the react render call on the client-side. Seems that all snapshot changes are somehow caused by this.

Reply:
Correct. I now scroll to the images, wait another second. This should be enough to have them reliably rendered in their final state.